backend/module/eCampApi/src/eCampApi/RpcConfigFactory.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace eCampApi;
+
+use eCamp\Lib\Entity\EntityLink;
+use eCamp\Lib\Entity\EntityLinkCollection;
+use Laminas\Di\Container\ServiceManager\AutowireFactory;
+
+class RpcConfigFactory {
```

Comment:
Could you rename and move this class to `\eCampApi\V1\Factory\RpcConfig` to match the existing `Config.php` from #827?

Reply:
Duplikat von https://github.com/ecamp/ecamp3/pull/1027#discussion_r584193101

Reply:
sure